src/main/java/uk/ac/cam/cl/dtg/segue/api/InfoFacade.java

```diff
@@ -42,10 +43,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import uk.ac.cam.cl.dtg.segue.auth.exceptions.NoUserLoggedInException;
```

Comment:
I don't think this is being used in this pull request.

Reply:
Will change now